Tests/ArcGISToolkitTests/PreplannedMapModelTests.swift

```diff
@@ -32,7 +32,7 @@ private extension PreplannedMapAreaProtocol {
 class PreplannedMapModelTests: XCTestCase {
     @MainActor
     func testInit() {
-        class MockPreplannedMapArea: PreplannedMapAreaProtocol {
+        final class MockPreplannedMapArea: PreplannedMapAreaProtocol {
```

Comment:
An alternative (that I would suggest) would be to turn this into a struct:

```swift
struct MockPreplannedMapArea: PreplannedMapAreaProtocol {
    func retryLoad() async throws { }
}
```

You could even move that `retryLoad()` implementation into the `PreplannedMapAreaProtocol` extension so that you don't need it in three separate places. The assertion would then look like this:

```swift
XCTAssert(model.preplannedMapArea is MockPreplannedMapArea)
```